src/script/builder/tir/for_frame.cc

```diff
@@ -92,6 +94,43 @@ ForFrame Grid(Array<PrimExpr> extents) {
 
 TVM_REGISTER_NODE_TYPE(ForFrameNode);
 
+TVM_REGISTER_GLOBAL("script.builder.tir.EnterForFrame")
+  .set_body_method<ForFrame>(&ForFrameNode::EnterWithScope);
+
+TVM_REGISTER_GLOBAL("script.builder.tir.ExitForFrame")
+  .set_body_method<ForFrame>(&ForFrameNode::ExitWithScope);
+
+
+TVM_REGISTER_GLOBAL("script.builder.tir.Serial")
+  .set_body_typed([](PrimExpr min, PrimExpr extent, Map<String, ObjectRef> attrs){
+    return Serial(min, extent, attrs);
+  });
```

Comment:
can we simplify it a bit to:

```suggestion
TVM_REGISTER_GLOBAL("script.builder.tir.Serial")
  .set_body_typed(Serial);
```